app/org/apache/spark/SparkMetricsAggregator.java

```diff
@@ -15,27 +15,69 @@
  */
 
 package org.apache.spark;
+import com.linkedin.drelephant.math.Statistics;
+import org.apache.commons.io.FileUtils;
 
 import com.linkedin.drelephant.analysis.HadoopApplicationData;
 import com.linkedin.drelephant.analysis.HadoopMetricsAggregator;
 import com.linkedin.drelephant.analysis.HadoopAggregatedData;
 import com.linkedin.drelephant.configurations.aggregator.AggregatorConfigurationData;
+import com.linkedin.drelephant.spark.data.SparkApplicationData;
+import com.linkedin.drelephant.spark.data.SparkExecutorData;
+import com.linkedin.drelephant.util.MemoryFormatUtils;
+import java.util.Iterator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 public class SparkMetricsAggregator implements HadoopMetricsAggregator {
 
+  private static final Logger logger = LoggerFactory.getLogger(SparkMetricsAggregator.class);
+
   private AggregatorConfigurationData _aggregatorConfigurationData;
 
+  private static final String SPARK_EXECUTOR_MEMORY = "spark.executor.memory";
+
+  private HadoopAggregatedData _hadoopAggregatedData = new HadoopAggregatedData();
+
+
   public SparkMetricsAggregator(AggregatorConfigurationData _aggregatorConfigurationData) {
     this._aggregatorConfigurationData = _aggregatorConfigurationData;
   }
 
   @Override
   public void aggregate(HadoopApplicationData data) {
+    long resourceUsed = 0;
```

Comment:
Can you update the Spark Metrics calculation logic in the wiki page?

https://github.com/linkedin/dr-elephant/wiki/Metrics-and-Heuristics#metrics